### PR TITLE
Fix 3 migration to VCD,  2 links in fr fr-ca tab

### PR DESCRIPTION
**Pull request by pg-ovh**

### CODE REVIEW

**Review by bs-ovh — CHANGES_REQUESTED** [review]
- Paths not fixed for: pages/bare_metal_cloud/managed_bare_metal/vmware_drs_distributed_resource_scheduler/
- Wrong format: "[xxx]   `(<xxx>)"` (68)

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.pt-pt.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ The table below lists each of the migration bottlenecks and criticality levels t
...
+| **Step 7**  | ⚠️           | **Resource pools (sharing)**            | No solutions for now.                                                                                                                                        | - Resource pools will be lost after migration because this notion no longer exists on the VCD side. We recommend the use of the vApp concepts in the VCD control panel.                                                | [Working with vApps in the VCD on OVHcloud control panel](<https://docs.vmware.com/en/VMware-Cloud-Director/10.5/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html>) |
```
Comment (bs-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.pt-pt.md]: Fix this link

**Review by bs-ovh — CHANGES_REQUESTED** [review]
- Fix ext. link format

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.pt-pt.md (hunk 1 above)
Comment (pg-ovh) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.pt-pt.md]: < https://docs.vmware.com/en/VMware-Cloud-Director/10.6/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html > -> https://docs.vmware.com/en/VMware-Cloud-Director/10.5/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html @tcpdumpfbacke ?

Comment (bs-ovh) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.pt-pt.md]: https://www.markdownguide.org/basic-syntax/#links

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.en-ie.md [hunk 2]
```diff
@@ -61,11 +61,11 @@ The table below lists each of the migration bottlenecks and criticality levels t
...
+| **Step 3**  | ️⚠️⚠️        | **DRS affinity/anti-affinity**          | Reconstitution of affinity/anti-affinity rules in VCD.                                                                                                       | - To be retained, DRS affinity/anti-affinity rules will have to be manually recreated by you in VCD on OVHcloud after migration.                                                                                       | [VMware DRS distributed resource scheduler)](/pages/bare_metal_cloud/managed_bare_metal/vmware_drs_distributed_ressource_scheduler)                                                                    |
```
Comment (bs-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.en-ie.md]: Complete brackets () or remove single bracket

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**